x-pack/test_serverless/functional/test_suites/observability/infra/header_menu.ts

```diff
@@ -1,38 +0,0 @@
-/*
```

Comment:
This test doesn't make sense anymore because the alert feature is now enabled on Serverless